src/app.js

```diff
@@ -36,7 +38,7 @@ const createApp = config => {
   app.use(ConfigErrorHandler);
   app.use(ReadonlyErrorHandler);
   app.use((err, req, res, next) => {
-    console.error(err.stack);
+    req.logger.error(err.stack);
```

Comment:
thought: The patch coverage for this PR is low and is not passing the CI. I believe one of the reasons is that it that it targets error handling codes from other PRs that were never covered: the scope of this PR overlaps with the technical debt of many others.

Probably it will be necessary to add tests increasing the coverage over the updated lines, increasing the original scope of this PR.

Reply:
The reason was that some tests were failing.
Once the issue was resolved the patch coverage came up to 87%